### PR TITLE
[RDAPI-29770] Updating default cipher string

### PR DESCRIPTION
**Pull request by Bcolemanaxway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Updating default cipher string to align with what is shown in Policy Studio.

## Deploy preview link
http://pr-2629.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_policydev/apigw_external_connections/common_icap_conf/#security-settings
http://pr-2629.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_policydev/apigw_gw_instances/general_services/#configure-advanced-ssl-settings
http://pr-2629.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_policydev/apigw_polref/routing_common/#advanced-ssl

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 7513e42 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-2629.axway-open-docs.opendocs-builder.pcloud.axway.int |

File: content/en/docs/apim_policydev/apigw_external_connections/common_icap_conf.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ tab. Select the check box next to the client certificate that you want to use to
 The **Ciphers**
 field enables you to specify the ciphers that API Gateway supports. API Gateway sends this list of supported ciphers to the destination ICAP server, which then selects the highest strength common cipher as part of the SSL handshake.
 The selected cipher is used to encrypt the data when it is sent over the secure channel.
-The default value is `FIPS:!SSLv3:!aNULL`.
+The default value is `FIPS:!SSLv3:!aNULL:!TLSv1:!kRSA`.
```
Comment (JohnD-Axway) [inline content/en/docs/apim_policydev/apigw_external_connections/common_icap_conf.md:54]: This isn't correct, the default value is still the old one.
Remove this update and we'll open a separate ticket to fix that.

**Review by JohnD-Axway — CHANGES_REQUESTED** [review]

File: content/en/docs/apim_policydev/apigw_external_connections/common_icap_conf.md (hunk 1 above)
Comment (JohnD-Axway) [reply content/en/docs/apim_policydev/apigw_external_connections/common_icap_conf.md:54]: Ticket created: https://jira.axway.com/browse/RDAPI-29874

Comment (Bcolemanaxway) [reply content/en/docs/apim_policydev/apigw_external_connections/common_icap_conf.md]: Reverted change to this file

**Review by JohnD-Axway — APPROVED** [review]

**Pull request merged**